backend/src/api.ts

```diff
@@ -295,114 +299,110 @@ loginCheckedGet('/signInPrompt/:id', async (req, _) => ({
-  await deleteTeamEventAttendance(req.body.uuid, user);
+loginCheckedDelete('/team-event/attendance/:uuid', async (req, user) => {
+  await deleteTeamEventAttendance(req.params.uuid, user);
   return {};
 });
 
 // Team Events Proof Image
-loginCheckedGet('/getEventProofImage/:name(*)', async (req, user) => ({
+loginCheckedGet('/event-proof-image/:name(*)', async (req, user) => ({
   url: await getEventProofImage(req.params.name, user)
 }));
-loginCheckedGet('/getEventProofImageSignedURL/:name(*)', async (req, user) => ({
+loginCheckedGet('/event-proof-image/:name(*)/signed-url', async (req, user) => ({
   url: await setEventProofImage(req.params.name, user)
 }));
-loginCheckedPost('/deleteEventProofImage', async (req, user) => {
-  await deleteEventProofImage(req.body.name, user);
+loginCheckedDelete('/event-proof-image/:name(*)', async (req, user) => {
+  await deleteEventProofImage(req.params.name, user);
   return {};
 });
 
 // Candidate Decider
-loginCheckedGet('/getAllCandidateDeciderInstances', async (_, user) => ({
+loginCheckedGet('/candidate-decider', async (_, user) => ({
   instances: await getAllCandidateDeciderInstances(user)
 }));
-loginCheckedGet('/getCandidateDeciderInstance/:uuid', async (req, user) => ({
+loginCheckedGet('/candidate-decider/:uuid', async (req, user) => ({
   instance: await getCandidateDeciderInstance(req.params.uuid, user)
 }));
-loginCheckedPost('/createNewCandidateDeciderInstance', async (req, user) => ({
+loginCheckedPost('/candider-decider', async (req, user) => ({
   instance: await createNewCandidateDeciderInstance(req.body, user)
 }));
-loginCheckedPost('/toggleCandidateDeciderInstance', async (req, user) =>
-  toggleCandidateDeciderInstance(req.body.uuid, user).then(() => ({}))
+loginCheckedPut('/candidate-decider/:uuid', async (req, user) =>
+  toggleCandidateDeciderInstance(req.params.uuid, user).then(() => ({}))
 );
-loginCheckedPost('/deleteCandidateDeciderInstance', async (req, user) =>
-  deleteCandidateDeciderInstance(req.body.uuid, user).then(() => ({}))
+loginCheckedDelete('/candidate-decider/:uuid', async (req, user) =>
+  deleteCandidateDeciderInstance(req.params.uuid, user).then(() => ({}))
 );
-loginCheckedPost('/updateCandidateDeciderRating', (req, user) =>
-  updateCandidateDeciderRating(user, req.body.uuid, req.body.id, req.body.rating).then(() => ({}))
+loginCheckedPut('/candidate-decider/:uuid/rating', (req, user) =>
+  updateCandidateDeciderRating(user, req.params.uuid, req.body.id, req.body.rating).then(() => ({}))
 );
-loginCheckedPost('/updateCandidateDeciderComment', (req, user) =>
-  updateCandidateDeciderComment(user, req.body.uuid, req.body.id, req.body.comment).then(() => ({}))
+loginCheckedPost('/candidate-decider/:uuid/comment', (req, user) =>
+  updateCandidateDeciderComment(user, req.params.uuid, req.body.id, req.body.comment).then(
+    () => ({})
+  )
 );
-loginCheckedPost('/sendMail', async (req, user) => ({
```

Comment:
We can't get rid of this endpoint. There are usages of it for email notifications.